editor/src/components/canvas/dom-walker.ts

```diff
@@ -1,6 +1,9 @@
 import * as React from 'react'
 import * as R from 'ramda'
 import { sides } from 'utopia-api'
+import * as ResizeObserverSyntheticDefault from 'resize-observer-polyfill'
+const ResizeObserver = ResizeObserverSyntheticDefault.default ?? ResizeObserverSyntheticDefault
```

Comment:
Oh this burns my eyes.

Reply:
I think the only solution is to cave in and enable synthetic default imports in our tsconfig.json

Reply:
Is there a reason why we haven't enabled it? I actually don't recall